### ISSUE TITLE
Question about AR model loss

### ISSUE THREAD

**Issue by asr-pub** [lifeiteng/vall-e#156]
Hello, I have a question about the CE loss, Why use the `sum` mode of CE loss ?

```python
    def forward(
        self,
        x: torch.Tensor,
        x_lens: torch.Tensor,
        y: Union[torch.Tensor, PromptedFeatures],
        y_lens: Union[torch.Tensor, PromptedFeatures],
        reduction: str = "sum",
        train_stage: int = 0,
        **kwargs,
    ) -> Tuple[torch.Tensor, Union[torch.Tensor, None]]:
  ...
  total_loss = F.cross_entropy(logits, targets, reduction=reduction)
  ...
```

**Comment by lifeiteng** [comment]
@asr-pub you can try `mean`
